kubernetes/config/storage.yaml

```diff
@@ -6,26 +6,65 @@ data:
     {
       blobstore: {
         contentAddressableStorage: {
-          circular: {
-            directory: '/cas',
-            offsetFileSizeBytes: 16 * 1024 * 1024,
-            offsetCacheSize: 10000,
-            dataFileSizeBytes: 10 * 1024 * 1024 * 1024,
-            dataAllocationChunkSizeBytes: 16 * 1024 * 1024,
+          'local': {
+            keyLocationMapOnBlockDevice: {
+              file: {
+                path: '/cas/key_location_map',
+                sizeBytes: 768 * 1024 * 1024,
+              },
+            },
+            keyLocationMapMaximumGetAttempts: 8,
+            keyLocationMapMaximumPutAttempts: 32,
+            oldBlocks: 8,
+            currentBlocks: 24,
+            newBlocks: 3,
+            blocksOnBlockDevice: {
+              source: {
+                file: {
+                  path: '/cas/blocks',
+                  sizeBytes: 10 * 1024 * 1024 * 1024,
+                },
+              },
+              spareBlocks: 3,
+            },
+            persistent: {
+              stateDirectoryPath: '/cas/persistent_state',
+              minimumEpochInterval: '300s',
+            },
           },
         },
         actionCache: {
-          circular: {
-            directory: '/ac',
-            offsetFileSizeBytes: 1024 * 1024,
-            offsetCacheSize: 1000,
-            dataFileSizeBytes: 100 * 1024 * 1024,
-            dataAllocationChunkSizeBytes: 1024 * 1024,
-            instances: ['bb-event-service', 'remote-execution'],
+          completenessChecking: {
```

Comment:
There should not be a "completenessChecking" here since there is already one specified at the frontend. This checking is only aware of the CAS of this single shard, and therefor will return some NotFounds for AC items that reference CAS items not allocated to this shard